### DISCUSSION TITLE
Ability to add relations to file uploads

### DISCUSSION THREAD

**Discussion by abanchev** [strapi/strapi#7973] [Legacy Topics]
I am uploading around 700k files to strapi, and after that it will be a nightmare to filter what is where. It would be very useful to be able to add relations (for example labels content type) so that later i can filter by type in the media picker. Any thoughts on how this can be implemented?

**Comment by derrickmehaffy** [comment]
Hello, we are already planning this, but it will require quite a bit of work. We estimate this to be added into the list view by the end of Q4 2020. Currently we don't have any workarounds or suggestions to add it yourself, as it requires quite a lot of code modification.
